closure/goog/testing/net/xhrio_test.js

```diff
@@ -55,6 +55,44 @@ function testStaticSend() {
   assertEquals('sendInstances_ after READY', 0, sendInstances.length);
 }
 
+function testStaticSendWithFactory() {
+  var xhrFactoryCalled = 0, optionsFactoryCalled = 0, callbackInvoked = 0;
+  var wrapperFactory = new goog.net.WrapperXmlHttpFactory(
+      function() {
+        xhrFactoryCalled++;
+        return goog.net.XmlHttp();
+      },
+      function() {
+        optionsFactoryCalled++;
+        return {};
+      });
+
+  sendInstances = goog.testing.net.XhrIo.getSendInstances();
+  var returnedXhr = goog.testing.net.XhrIo.send(
+      'url', undefined, 'GET', '', undefined, 0, false, wrapperFactory);
```

Comment:
I'd prefer `null` instead of `undefined` see also a [call to goog.net.XhrIo.send](https://github.com/google/closure-library/blob/e877b1eac410c0d842bcda118689759512e0e26f/closure/goog/debug/errorreporter.js#L221).